app/src/test/java/org/fossasia/openevent/app/presenter/LoginPresenterTest.java

```diff
@@ -136,4 +144,12 @@ public void shouldNotAccessView() {
         Mockito.verifyNoMoreInteractions(loginView);
     }
 
+    @Test
+    public void shouldGetEmailListSuccessfully() {
```

Comment:
This test is actually like testing getter and setter. We test flow of information among view and presenter